src/chains-config/westendControllers.ts

```diff
@@ -24,6 +24,7 @@ export const westendControllers: ControllerConfig = {
 		TransactionSubmit: true,
 		PalletsStakingProgress: true,
 		PalletsStorage: true,
+		Paras: true,
```

Comment:
Yes. Or more specifically we mark it `true` if we want to mount the `ParasController`, which is only relevant if you have the auctions, crowdloans, and slots module from the polkadot repo.

Reply:
The thought I had was if, as it's a property of relay chains to have parachains, we could name this `isRelayChain` or some such. But `paras` seems more straight forward.

Reply:
> only relevant if you have the auctions, crowdloans, and slots module from the polkadot repo.

Would it be possible to derive the `hasParas` property by detecting the presence of these modules?

Reply:
> Would it be possible to derive the hasParas property by detecting the presence of these modules?

Yes, this is definitely possible but we choose not do it because I believe it is not a safe assumption to mount a controller based on detected pallets. 

Prior to integrating non parity chains to sidecar my idea was to do just this with all the endpoints: detect if the necessary pallets existed and if they did mount the controller.

However I quickly encountered a lot of complexity because pallets are typically used in tight conjunction with other pallets, can be instantiated in vastly varying ways etc; and so we end up needing to make some assumptions about how the pallets are configured in the individual runtimes. Because of this I decided to go with this explicit, manual configuration of which controllers to mount because at the end of the day we need a human (parachain builder) to ensure the assumptions that the controllers & services make hold true for the particular chain

Reply:
That makes sense, ty.